sandbox.config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "infiniteLoopProtection": true,
+    "hardReloadOnChange": true,
+    "view": "browser",
+    "template": "node"
+}
```

Comment:
can you generate it in the script instead?

Reply:
Yh sure let me do that